sherpa/astro/ui/utils.py

```diff
@@ -8635,18 +8637,25 @@ def fake_pha(self, id, arf, rmf, exposure, backscal=None, areascal=None,
            dataset identifier is used, rather than creating a
            completely new dataset.
 
+        .. versionchanged:: 4.13.2
+           Multiple ARFs and RMFs can now be used for the simulation.
+           Furthermore, background models can be specified instead of
+           background `~sherpa.stro.data.DataPHA` datasets.
```

Comment:
```suggestion
           background `~sherpa.astro.data.DataPHA` datasets.
```

I am somewhat loathe to add too many sphinx-isms to the documentation as I find them distracting and, for the occasional user, rather baffling (e.g. I know in the past I've looked at numpy and/or matplotlib docs and had to spend time carefully reading the docs). Of course, I've also added `.. versionchanged` lines which are the same thing pand have at least one header for a module which uses this very `~xxx` syntax], but they are a bit-more obvious and I can easily handle them when I create ahelp files, which is less the case for things like `~sherpa.astro.data.DataPHA`.  So I'm not saying no to this, just that we should think about this a bit.

Reply:
"Do we want links in sphinx" is a bigger question than this PR. We should sit down at some point and think about what we want to do. Personally, I find them very useful in the rendered sphinx docs and would advocate to use them in many more places ;-) but I don't know what implications that has for generating ahelp files, if this markup should be stripped etc. #1133 could be the place to discuss that. (Just as a short note: For the marx docs, I've written a [custom sphinx extention](https://github.com/Chandra-MARX/doc/blob/main/source/marxmarkup.py) to make use of the sphinx mark-up for marx parameters values and to generate links to ahelp pages. I imagine that we could do a similar thing here if we need it.)